### DISCUSSION TITLE
Do this support recursion ?

### DISCUSSION THREAD

**Discussion by opticaloptical** [laike9m/Cyberbrain#128] [Q&A] [answered]
I try it, but it dosn't  work.
![无标题](https://user-images.githubusercontent.com/6635193/121773640-c0d79000-cbaf-11eb-91a1-b9a315aeab3d.png)

**Comment by laike9m** [comment, marked as answer]
Support for recursion is limited at this point. The program won't crash, but you won't be able to see meaningful results.

**Reply by opticaloptical** [reply]
thanks

**Reply by opticaloptical** [reply]
Are there any plans to support it？

**Reply by laike9m** [reply]
Eventually yes, but not in the near future
